### ISSUE TITLE
Add support for reading and writing polylines

### ISSUE THREAD

**Issue by frewsxcv** [georust/geozero#215]
https://github.com/georust/polyline

**Comment by michaelkirk** [comment]
Polyline is a bit different from the other geometry formats we've supported thus far. Primarily it's only for encoding a single line string. 

Some outstanding questions:

1. Do we handle multi line strings? (and how?)
2. What do we do for other geometries?

**Comment by frewsxcv** [comment]
Agreed the writing behavior would need to be discussed, but (I think?) the reading behavior seems pretty straightforward. Read the string, write linestring(s) as the output.
